types/validation_test.go

```diff
@@ -186,9 +205,11 @@ func TestValidatorSet_VerifyCommitLight_ReturnsAsSoonAsMajorityOfVotingPowerSign
 
 	err = valSet.VerifyCommitLight(chainID, blockID, h, commit)
 	assert.NoError(t, err)
+	err = valSet.VerifyCommitLightAllSignatures(chainID, blockID, h, commit)
```

Comment:
nit: Maybe rename `[...]_ReturnsAsSoonAsMajorityOfVotingPowerSigned` as well because it checks something else as well.

Reply:
Good catch